src/lib/operators/window_function_evaluator/util.hpp

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <array>
+#include <compare>  // NOLINT(build/include_order)
+#include <span>     // NOLINT(build/include_order)
```

Comment:
Add TODOs so that when one day the linter knows about C++20, the `NOLINT` s can be removed.

Reply:
We decided to not add TODOs for `//NOLINT ` (see argument from previous comment).